packages/babel-plugin-transform-es2015-modules-amd/src/index.js

```diff
@@ -89,7 +82,6 @@ export default function({ types: t }, options) {
               MODULE_NAME: moduleName,
 
               AMD_ARGUMENTS: t.arrayExpression(amdArgs),
-              COMMONJS_ARGUMENTS: commonjsArgs,
```

Comment:
The nice thing about adding validation is that it catches all you're dumb mistakes :D